src/puripuly_heart/core/stt/controller.py

```
@@ -307,7 +307,9 @@ async def _on_speech_end(self, event: SpeechEnd) -> None:
                 fallback_level=logging.INFO,
             )
             self._emit_stt_input_diagnostics(event.utterance_id, finalize=True)
-            await self._active_session.on_speech_end(trailing_silence_ms=event.trailing_silence_ms)
+            asyncio.create_task(
+                self._active_session.on_speech_end(trailing_silence_ms=event.trailing_silence_ms)
+            )
```

Ruff (0.15.21)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)